scripts/changelog/collect-package-changelogs.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -e
+[[ $DEBUG ]] && set -x
+
+if [ -z "${IMAGE_UUID}" ] ; then
+  echo "IMAGE_UUID env variable is mandatory"
+  exit 0
```

Comment:
It seems to be error case so return code should not be 0.

Reply:
done